speculation-rules.bs

```diff
@@ -649,6 +649,27 @@ The same mitigations available to other features also apply here. In particular,
 
 The possibility of leaking link URLs via this mechanism is additionally mitigated by the fact that prefetch and prerender to plaintext HTTP (other than to localhost) is not permitted, and so such an on-path attacker could not directly observe preloading request URLs, but would only have access to metadata and traffic analysis. This does not, however, replace standard XSS protections.
 
+<h3 id="security-inline-scripting">Inline scripting</h3>
```

Comment:
I don't think this belongs in the Security Considerations section, which is mostly "here's how we thought about security when we wrote this specification". We could certainly mention this in the section on XSS (Cross-site scripting), but I believe the normative patches to CSP should be in an earlier section. I'd suggest making it a top-level section called something like "CSP Patches", between the Speculation Rules and Security Considerations sections.